src/ape/__init__.py

```diff
@@ -4,93 +4,23 @@
 
 import sys as _sys
 from functools import partial as _partial
-from pathlib import Path as _Path
 
 from .contracts import _Contract
-from .managers.accounts import AccountManager as _AccountManager
-from .managers.chain import ChainManager as _ChainManager
-from .managers.compilers import CompilerManager as _CompilerManager
-from .managers.config import ConfigManager as _ConfigManager
-from .managers.converters import ConversionManager as _ConversionManager
-from .managers.networks import NetworkManager as _NetworkManager
-from .managers.project import ProjectManager as Project
-from .plugins import PluginManager as _PluginManager
-from .utils import USER_AGENT
-
-# Wiring together the application
-
-plugin_manager = _PluginManager()
-"""Manages plugins for the current project. See :class:`ape.plugins.PluginManager`."""
-
-_ConfigManager.plugin_manager = plugin_manager
-config = _ConfigManager(
-    # Store all globally-cached files
-    data_folder=_Path.home().joinpath(".ape"),
-    # NOTE: For all HTTP requests we make
-    request_header={
-        "User-Agent": USER_AGENT,
-    },
-    # What we are considering to be the starting project directory
-    project_folder=_Path.cwd(),
-)
-"""The active configs for the current project. See :class:`ape.managers.config.ConfigManager`."""
-
-# Main types we export for the user
-
-_CompilerManager.config = config
-_CompilerManager.plugin_manager = plugin_manager
-compilers = _CompilerManager()
-"""Manages compilers for the current project. See
-:class:`ape.managers.compilers.CompilerManager`."""
-
-_NetworkManager.config = config
-_NetworkManager.plugin_manager = plugin_manager
-networks = _NetworkManager()
-"""Manages the networks for the current project. See
-:class:`ape.managers.networks.NetworkManager`."""
-
-_ConversionManager.config = config
-_ConversionManager.plugin_manager = plugin_manager
-_ConversionManager.networks = networks
-_converters = _ConversionManager()
-
-_ChainManager._networks = networks
-_ChainManager._converters = _converters
-chain = _ChainManager()
-"""
-The current connected blockchain; requires an active provider.
-Useful for development purposes, such as controlling the state of the blockchain.
-Also handy for querying data about the chain and managing local caches.
-"""
-
-_AccountManager.config = config
-_AccountManager.converters = _converters
-_AccountManager.plugin_manager = plugin_manager
-_AccountManager.network_manager = networks
-accounts = _AccountManager()
-"""Manages accounts for the current project. See :class:`ape.managers.accounts.AccountManager`."""
-
-Project.config = config
-Project.compilers = compilers
-Project.networks = networks
-Project.converter = _converters
-"""User-facing class for instantiating Projects (in addition to the currently
-active ``project``). See :class:`ape.managers.project.ProjectManager`."""
+from .managers import Project, accounts, chain, config, converters, networks
```

Comment:
meant to say this before, but too much going on, wanted `converters` to be imported private

```suggestion
from .managers import Project, accounts, chain, config, networks
from .managers import converters as _converters
```